src/ray/core_worker/core_worker.cc

```diff
@@ -1896,7 +1896,8 @@ std::vector<rpc::ObjectReference> CoreWorker::SubmitTask(
   auto constrained_resources =
       AddPlacementGroupConstraint(task_options.resources, scheduling_strategy);
 
-  const std::unordered_map<std::string, double> required_resources;
+  const std::unordered_map<std::string, double> required_resources =
```

Comment:
set inside task_options instead of here?

Reply:
task options has one field...